prbench/tests/envs/tidybot/test_tidybot3d_utils.py

```diff
@@ -0,0 +1,364 @@
+"""Tests for TidyBot utility functions."""
+
+import numpy as np
+import pytest
+
+from prbench.envs.tidybot.utils import (
+    bboxes_overlap,
+    get_table_bbox,
+    sample_collision_free_position,
+    sample_pose_in_region,
+)
+
+
+class TestGetTableBbox:
```

Comment:
we typically don't put the tests inside classes, so let's remove that for consistency unless we need the classes for some reason